src/util_defs.h

```diff
@@ -0,0 +1,12 @@
+#ifndef SECP256K1_UTIL_DEFS_H
+#define SECP256K1_UTIL_DEFS_H
```

Comment:
Nit: `util_defs.h` is pretty ambiguous. How about `util_local_visibility.h` or so?

Reply:
Thanks! Reworked.